ztp/extra-manifests-builder/99-one-shot-time-sync/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+MCPROLE=${MCPROLE:-master}
+sync_time_once_base64=$(base64 -w0 sync-time-once.sh)
+
+
+echo "\
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: ${MCPROLE}
+  name: 99-one-shot-time-sync-${MCPROLE}
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,${sync_time_once_base64}
+          mode: 493
+          overwrite: true
+          path: /usr/local/bin/sync-time-once.sh
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=Sync time once
+            After=network.service
+            [Service]
+            Type=oneshot
+            TimeoutStartSec=infinity
+            ExecStart=/usr/local/bin/sync-time-once.sh
```

Comment:
Can we change this to 
```suggestion
            Environment=SYNC_ATTEMPT_TIMEOUT_SEC=300
            ExecStart=/usr/sbin/chronyd -n -f /etc/chrony.conf -q -t $SYNC_ATTEMPT_TIMEOUT_SEC
```
and eliminate the script?

Reply:
Done